### ISSUE TITLE
Outdated Hunter distribution

### ISSUE THREAD

**Issue by hiptmair** [craffael/lehrfempp#146]
The LehrFEM++ hunter distribution should be updated from the current master branch, because the signature of some functions has changed in the meantime.

An example is `lf::uscalfe::NodalProjection`.

We urgently need this update because of the forthcoming examination.

**Comment by hiptmair** [comment]
The big problem is that the online documentation reflects the current implementation in the master branch.

**Comment by craffael** [comment]
Okay I've prepared a new release which takes a bit of time because many (hunter) tests have to run through. At the moment I'm waiting for the developper of Hunter (ruslan baratov) to merge my [PR](https://github.com/ruslo/hunter/pull/1927). As soon as this is done the newest version can be used. Usually it takes about 1-24h. However Ruslan is playing with the thought [to stop maintaining hunter on August 25th](https://github.com/ruslo/hunter/issues/1921) so it could be that it takes longer this time. 

@hiptmair until when do you need the new release? Alternatively we could also put the documentation of the last release (0.7.19) online.

**Comment by hiptmair** [comment]
Thanks a lot, if the new distribution is available by Friday, then this should be sufficient. How can I find out whether the new version is ready for download?

**Comment by craffael** [comment]
As soon as my [PR](https://github.com/ruslo/hunter/pull/1927) is merged into Hunter, we can upgrade to the new version. If you want, you can subscribe to that PR and then you should be informed once it is merged.
I will also update the LehrFEM++ getting started guide with the new version so I can also inform you when the new version is availabile.

**Comment by hiptmair** [comment]
The new version has been merged into hunter 
`HunterGate(
    URL "https://github.com/ruslo/hunter/archive/v0.23.207.tar.gz"
    SHA1 "6e1b05a1f65c2aa492f7f43a43578e83ac62bbdd"
)`

**Comment by hiptmair** [comment]
I was wrong, the allegedly new Hunter distribution still contains the old version:

In master branch in [fe_tools.h](https://github.com/craffael/lehrfempp/blob/master/lib/lf/uscalfe/fe_tools.h):
~~~
template <typename SCALAR, typename MF, typename SELECTOR = base::PredicateTrue>
auto NodalProjection(const UniformScalarFESpace<SCALAR> &fe_space, MF &&u,
                     SELECTOR &&pred = base::PredicateTrue{}) {
~~~
 In `.hunter/_Base/6e1b05a/696e52c/3e02db1/Install/include/lf/uscalfe`:
~~~
template <typename SCALAR, typename FUNCTOR,
          typename SELECTOR = base::PredicateTrue>
auto NodalProjection(std::shared_ptr<UniformScalarFESpace<SCALAR>> fe_space,
                     FUNCTOR &&u, SELECTOR &&pred = base::PredicateTrue{}) {
~~~

The file in the Hunter distribution also seems to be rather old:
~~~
uscalfe $ ls -al fe_tools.h 
-rw-------. 2 ralfh p-sam 14889 Mar 20 11:52 fe_tools.h
uscalfe $ pwd
/scratch/users/ralfh/.hunter/_Base/6e1b05a/696e52c/3e02db1/Install/include/lf/uscalfe
~~~
